Sources/ClientRuntime/Networking/Http/HTTPResponse+WireDataProviding.swift

```diff
@@ -7,9 +7,10 @@
 
 import class SmithyHTTPAPI.HTTPResponse
```

Comment:
Below you will see that many declararions and imports of those declarations are tagged with `@_spi(...)` to prevent them from showing up as public API.

I will not comment on the rest of these changes below.